apis/python/src/tiledbsoma/io/outgest.py

```diff
@@ -237,6 +256,7 @@ def to_anndata(
     var_id_name: str | None = None,
     obsm_varm_width_hints: dict[str, dict[str, int]] | None = None,
     uns_keys: Sequence[str] | None = None,
+    dask: DaskConfig | None = None,
```

Comment:
lifecycle tag -- experimental

Reply:
also, seems to be entirely missing docstring for this param

Reply:
I added a sentence about it in the docstr, and tried to indicate that that param specifically is `lifecycle: experimental`. lmk what you think.